tests/symbolicator/snapshots/SymbolicatorUnrealIntegrationTest/test_unreal_crash_with_attachments.pysnap

```diff
@@ -104,167 +104,9 @@ exception:
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff754814e4c'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff754805258'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff7571fcd39'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff75739984f'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff75739082f'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff757aafb58'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff757aa121d'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff754d8cf00'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff754d8cc56'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff757a56186'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff757a3e77e'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff756f2f984'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff756f06dd3'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff756cff2ee'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
-      - in_app: false
-        instruction_addr: '0x7ff754be3394'
-        package: \\Mac\Home\Desktop\WindowsNoEditor\YetAnother\Binaries\Win64\YetAnother.exe
-        trust: prewalked
+      type: minidump
     stacktrace:
       frames:
-      - in_app: false
```

Comment:
This diff is OK. Since the crash is processed without symbols, this hits an edge case in the FP recovery in the amd64 stackwalker in Breakpad. If this turns out to be an issue to some customers, we can fix this in Breakpad.